src/events/eventTypes.ts

```diff
@@ -112,13 +127,15 @@ export const fromDjsArgs = {
 };
 /* eslint-enable @typescript-eslint/explicit-module-boundary-types */
 
+/*
 // Validate the above types against DJS types
 type ValidFromDjsArgs = {
   [key in keyof ExtendedClientEvents]: (...args: ExtendedClientEvents[key]) => unknown;
 };
 type AssertEquals<TActual, TExpected> = TActual extends TExpected ? true : false;
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const fromDjsArgsIsValid: AssertEquals<typeof fromDjsArgs, ValidFromDjsArgs> = true;
+*/
```

Comment:
These are intended to validate that our types match DJS types during TS compilation. Was there an issue that they didn't work anymore?

Reply:
*Talked about in Discord*